docs/AllelicCapSeg/ACS-methods.tex

```diff
@@ -315,89 +293,176 @@ \subsection{Bayesian het pulldown} \label{bayesian-het-pulldown}
+\end{equation}
+%
+As before, the ref minor case is obtained from $f \leftrightarrow 1 - f$.  We present the complete initialization process in Algorithm \ref{initialization}.
 
-Since $\alpha$ and $\beta$ are correlated, naively alternating Gibbs sampling of $\alpha$ and $\beta$ may be slow.  A natural choice of variables is the mean $\mu$ and standard deviation $\sigma$ (of ${\rm Gamma}(\alpha, \beta)$, in terms of which $\alpha = \mu^2/\sigma^2$ and $\beta = \mu/\sigma^2$.  Intuitively these ought to be weakly-coupled.  Empirically we find that the conditionals on $\mu$ and $\sigma$ are always unimodal\footnote{For $\mu$ this is actually pretty simple to show analytically.} and thus sampling is straightforward.  Slice sampling is a reasonable method to use.
+\begin{algorithm}
+\begin{algorithmic}[1]
+\State Set $\bar{z}_{ja} = 1$ if $a_j < r_j$, else $\bar{z}_{jr} = 1$.
+\State \textproc{initializeAlleleFraction}
+\State Set $\pi = 0.01$
+\State Set $\mu = 1$
+\State Set $\beta = 10$
 
-\subsection{Model-comparison test for segment merging} \label{likelihood-based-segment-merging}
+\Statex
 
-\SL{I'll fill this in once it's worked out.}
+\Repeat
+	\State \textproc{calculateResponsibilities}
+	\State \textproc{initializeAlleleFraction}
+	\State \textproc{initializeOutlierProbability}
+	\State \textproc{initializeGammaHyperparameters}
+\Until{a low standard of convergence}
 
-%%%APPENDICES
-\appendix
+\Statex
+
+\Procedure{calculateResponsibilities}{}
+  	\State $(p_{ja}, p_{jr}, p_{jo}) = \left(  (1-\pi)\phi(\alpha, \beta, f_j, a_j, r_j) , (1-\pi)\phi(\alpha, \beta, 1-f_j, a_j, r_j) ,  \frac{2 \pi a_j! r_j!}{(n_j + 1)!} \frac{\Gamma(\alpha)}{\beta^\alpha}  \right)$ for each $j$.
+	\State $\left( \bar{z}_{ja}, \bar{z}_{jr}, \bar{z}_{jo} \right) = (p_{ja}, p_{jr}, r_{jo}) / (p_{ja} + p_{jr} + p_{jo}) $ for each $j$
+\EndProcedure
+
+\Statex
+
+\Procedure{initializeAlleleFraction}{}
+  	\State Set $f_S = \sum_{j \in S} \left( \bar{z}_{ja} a_j + \bar{z}_{jr} r_j \right) / \sum_{j \in S} \left( \bar{z}_{ja} + \bar{z}_{jr} \right) $ for each $S$
+\EndProcedure
+
+\Statex
+
+\Procedure{initializeOutlierProbability}{}
+	\State Set $\pi = \sum_j \bar{z}_{jo} / N$
+\EndProcedure
+
+\Statex
+
+\Procedure{initializeGammaHyperparameters}{}
+	\State $\ave{\lambda^k_j}_a = \phi(\alpha + k, \beta, f_j, a_j, r_j) / \phi(\alpha, \beta, f, a, r)$ for $k = 1, \, 2$ for all $j$
+	\State $\ave{\lambda^k_j}_r = \phi(\alpha + k, \beta, 1 - f_j, a_j, r_j) / \phi(\alpha, \beta, 1 - f_j, a_j, r_j)$ for $k = 1, \, 2$ for all $j$
+	\State $\ave{\lambda^k} = \sum_j \left( \bar{z}_{ja} \ave{\lambda^k_j}_a +  \bar{z}_{jr} \ave{\lambda^k_j}_r \right) / \sum_j \left( \bar{z}_{ja} +  \bar{z}_{jr} \right)$ for $k = 1, \, 2$
+	\State Set $\mu = \ave{\lambda}$
+	\State Set $\beta = \ave{\lambda}/ \left(\ave{\lambda^2} - \ave{\lambda}^2 \right)$
+\EndProcedure
+
+\end{algorithmic}
+\caption{Initializing the \ACS ~ Model}
+\label{initialization}
+\end{algorithm}
```

Comment:
PDF reads fine other than noted comments. will go back and check math later as discussed.